fdbclient/ClientKnobs.cpp

```diff
@@ -303,7 +304,7 @@ void ClientKnobs::initialize(Randomize randomize) {
 
 	init( ENABLE_ENCRYPTION_CPU_TIME_LOGGING,       false );
 	init( SIMULATION_EKP_TENANT_IDS_TO_DROP,         "-1" );
-	init( ENABLE_CONFIGURABLE_ENCRYPTION,           false );
+	init( ENABLE_CONFIGURABLE_ENCRYPTION,            true ); if ( randomize && BUGGIFY ) ENABLE_CONFIGURABLE_ENCRYPTION = !ENABLE_CONFIGURABLE_ENCRYPTION;
```

Comment:
I'm against buggifying this knob. it will create a pain later on to maintain the restart tests. say we keep the knob and  buggify it in 7.3.0, and remove it completely in 7.3.1. The new code will not recognize the knob, while with the old code we will need to disable the knob. We will need to create a whole new copy of every restart tests for the two versions.

Reply:
Sounds good to me. I will keep the `knob` enabled for now.